src/Client/InfoObtainingClient/InfoClient.php

```diff
@@ -537,6 +567,28 @@ private function readPictureFromCurrentDC(
         });
     }
 
+    public function warmup(): void
```

Comment:
Specify on interface level
All public methods must be declared on interface-level

Reply:
Fixed, added several interfaces